fetch.bs

```diff
@@ -1870,6 +1881,17 @@ is a <a>filtered response</a> whose
  <li><p>Return <var>newResponse</var>.
 </ol>
 
+<p>A <dfn id=concept-fresh-response>fresh response</dfn> is a <a for=/>response</a> whose age is
+within its <a href=https://tools.ietf.org/html/rfc7234#section-4.2.1>freshness lifetime</a>.
+
+<p>A <dfn id=concept-stale-while-revalidate-response>stale-while-revalidate response</dfn> is a
+<a for=/>response</a> that is not a <a>fresh response</a>, and its current age is within
+the <a href=https://tools.ietf.org/html/rfc5861#section-3>stale-while-revalidate lifetime</a>.
```

Comment:
This reference doesn't define this term, which is a bit confusing.

Reply:
Do you have a better suggestion? I linked it to the closest definition but I don't necessarily want to rewrite that spec.

Reply:
I'm not sure, it's not really my area of expertise. I guess we can leave it for now...